testing/charts/test_boxplot_chart.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+from widgetastic.widget import View
+
+from widgetastic_patternfly5 import BoxPlotChart
+
+TESTING_PAGE_URL = "https://patternfly-react-main.surge.sh/charts/box-plot-chart"
+
+TEST_DATA = {
+    "2015": {"Cats": "q1: 1.75, q3: 3.5", "Limit": "12"},
+    "2016": {"Cats": "q1: 2.75, q3: 8.5", "Limit": "12"},
+    "2017": {"Cats": "q1: 4.25, q3: 6.5", "Limit": "12"},
+    "2018": {"Cats": "q1: 1.75, q3: 4.5", "Limit": "12"},
+}
+
+
+@pytest.fixture
+def view(browser):
+    class TestView(View):
+        ROOT = ".//div[@id='ws-react-c-box-plot-chart-embedded-legend']"
+        chart = BoxPlotChart(locator=".//div[@class='pf-v5-c-chart']")
+
+    return TestView(browser)
+
+
+def test_line_chart(view):
```

Comment:
Please change name of test to `test_boxplot_chart` ^^

Reply:
Done